src/javascript/app/base/binary_loader.js

```diff
@@ -74,14 +75,15 @@ const BinaryLoader = (() => {
         ContentVisibility.init();
 
         BinarySocket.wait('authorize', 'website_status', 'landing_company').then(() => {
-            const images = $('#content img');
-            if (images) {
-                images.on('load', () => {
+            // first time load.
+            const last_image = $('#content img').last();
```

Comment:
this still doesn't tell us if the image has loaded, or if it's really the last one, right?